R/score_model_out.R

```diff
@@ -8,6 +8,16 @@
 #' @param metrics Character vector of scoring metrics to compute. If `NULL`
 #' (the default), appropriate metrics are chosen automatically. See details
 #' for more.
+#' @param relative_metrics Character vector of scoring metrics for which to
+#' compute relative skill scores. The `relative_metrics` should be a subset of
+#' `metrics` and should only include proper scores (e.g., it should not contain
+#' interval coverage metrics).  If `NULL` (the default), no relative metrics
+#' will be computed.  Relative metrics are only computed if `summarize = TRUE`,
+#' and require that `"model_id"` is included in `by`.
```

Comment:
I think the alternative to enforcing this requirement is that we add another argument along the lines of `compare` as is used in `scoringutils::get_pairwise_comparisons`.  However, since essentially all use cases of this function will include `"model_id"` in `by`, I don't think it's necessarily worth introducing the extra argument here?

Reply:
I think the current approach is fine since it would be caught by the validation. The problem with extra arguments that affect other arguments is that it becomes difficult for users to remember the relationships between them.